_glaredb/connections-and-integrations/fabric.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Fabric
+parent: Connections and integrations
+---
+
+# Fabric
+
+[Microsoft Fabric] provides a suite of data analytics, data engineering, and BI
+tools on the Azure platform.
+
+In this guide, we'll walk through how to install and run GlareDB using a
+Notebook in Fabric.
+
+## Running GlareDB in a notebook
+
+The first step to running GlareDB in Fabric is to create a notebook. A notebook
+can be created by selecting **Notebook (Preview)** from the **Create** tab.
+
+![create]
+
+Once created, you'll be dropped into the notebook where you can begin execution
+python cells.
+
+Now we're able to install the [glaredb python library] by execution the
+following in a python cell:
+
+```text
+!pip install glaredb
+```
+
+If everything install successfully, you should now be able to import and run
+glaredb:
+
+```python
+import glaredb
+
+con = glaredb.connect()
+con.sql("select 'hello from fabric'").show()
+```
+
+And this is what the output should look like:
+
+![success]
+
+## Connecting to GlareDB Cloud from Fabric
+
+Connecting to GlareDB Cloud from Fabric is easy once you've created a deployment
+on GlareDB Cloud. To get connection details for the deployment, click the
```

Comment:
I'd href all mentions of `GlareDB Cloud` to https://console.glaredb.com

Reply:
Wording suggestion:

Connecting to [GlareDB Cloud](https://console.glaredb.com/) to your deployment is easy. If you're
a new user, a deployment is automatically created for you. If you have multiple, select the one
that you'd like to connect with. To get connection details ....